Python/pywarpx/picmi.py

```diff
@@ -739,6 +739,41 @@ def distribution_initialize_inputs(
             species.add_new_group_attr(source_name, "density", density_scale)
 
 
+class AnalyticFluxDistribution(
```

Comment:
Is there a different way to handle this? This is an exact duplicate of `UniformFluxDistribution` except for the one keyword argument `flux_expression`.

Reply:
Good point. 

Note that we also have two different classes for the volume injection: `UniformDistribution` and `AnalyticDistribution`, so we were trying to mirror the same setup here for `UniformFluxDistribution` and `AnalyticFluxDistribution`. 

Do you think that your comment also applies to `UniformDistribution` and `AnalyticDistribution`?

Reply:
Thanks for pointing this out, I will refactor accordingly.

Reply:
The UniformDistribution and AnalyticDistribution are quite a bit different from each other and it's probably too late to change that API.

Reply:
Is this PR ready? I still see the duplication of 'UniformFluxDistribution' - this is fine if that is the intent. Also, `UniformFluxDistribution` has the `inject_from_embedded_boundary` option - would that also be supported with the analytic distribution?